repo/packages/E/elastic/800/resource.json

```diff
@@ -0,0 +1,59 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz",
+      "bootstrap-zip": "https://downloads.mesosphere.com/elastic/assets/3.0.0-7.3.2/bootstrap.zip",
+      "scheduler-zip": "https://downloads.mesosphere.com/elastic/assets/3.0.0-7.3.2/elastic-scheduler.zip",
+      "elasticsearch-jre-tar-gz": "https://downloads.mesosphere.com/elastic/assets/openjdk-11.0.2_linux-x64_bin.tar.gz",
```

Comment:
is the FW here using jdk11 and the [scheduler is using jdk8](https://github.com/mesosphere/universe/pull/2396/files#diff-134733fc7a4bf8e4b4568cc5cbe869c0R4)?

Seems like Elastic is doing something a little different than the other FWs. Compare [to HDFS](https://github.com/mesosphere/universe/pull/2399/files#diff-6613aab3719e68ff0baee24598076289R4-R8) and dcos-common's [package_builder](https://github.com/mesosphere/dcos-commons/blob/master/tools/universe/package_builder.py#L20-L22). I wonder if the tooling is mixed up? 

Anyway, if the package works then it works. Just something I noticed

Reply:
Elasticsearch 7 requires JDK 11. The tooling isn't mixed up, the change to allow the Elasticsearch process to use a different JDK than the scheduler was made in the PR that updated the base-tech to Elasticsearch 7.
- https://github.com/mesosphere/dcos-elastic-service/commit/3935cfdc4c546952d4c789712d99f53faa93107e#diff-bd4b155ddd45979d6c1c59abcb5fea8aR5
- https://github.com/mesosphere/dcos-elastic-service/commit/3935cfdc4c546952d4c789712d99f53faa93107e#diff-df8798ae6864ea0c67b8f46c8ddc0175R8

Reply:
Copy that.

In that case, is the package actually using SDK `0.57.1` as reported [here](https://github.com/mesosphere/dcos-docs-site/pull/2565/files#diff-abd872aaacce61d4b9bd284187699622R24)? [0.57.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1) bumps the scheduler SDK to JDK11 too.

Reply:
Looking at the [release branch](https://github.com/mesosphere/dcos-elastic-service/tree/elastic-3.0.0-7.3.2-stable), seems like the SDK version was set back to `0.56.3`? That means it won't get all the new DCOS 2.0 features

Reply:
Yes, there were [issues with building the scheduler](https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=2326258&_focus=332) with the new SDK and we decided to revert to 0.56.3 to not delay the Elastic release.

Reply:
Bummer - okay let's get out the release as is; at least then we can work on a future release with a much smaller update and less moving parts